src/AndroidDebugLauncher/Telemetry.cs

```diff
@@ -40,6 +40,11 @@ public enum LaunchFailureCode
             RunAsFailure,
 
             /// <summary>
+            /// run-as returned 'Package 'bla' is not debuggable'
```

Comment:
I just used the same bogus example as RunAsPackageUnknown that was already there.